src/api/PDFDocument.ts

```diff
@@ -792,7 +792,7 @@ export default class PDFDocument {
     font: StandardFonts | string | Uint8Array | ArrayBuffer,
     options: EmbedFontOptions = {},
   ): Promise<PDFFont> {
-    const { subset = false } = options;
+    const { subset = false, customFontName = '' } = options;
```

Comment:
1. Can we remove the default value for this options? I'd prefer to pass it down as `undefined` since the embedders support that type.
2. Can we rename this to `customName`?